src/components/TileEditor.tsx

```diff
@@ -1,14 +1,78 @@
 import React from 'react';
-import { TileInfo } from '../firebase';
+import { firestore, TileInfo } from '../firebase';
+import { useForm } from "react-hook-form";
 
 export function TileEditor(props: {tileInfo: TileInfo, tileName: string}) {
+  const { register, setValue, handleSubmit, errors } = useForm<TileInfo>();
+  const onSubmit = handleSubmit( async ({ bg_color, bg_image, hover_text, description_text, link }) => {
+    const tile = firestore.collection("tiles").doc(props.tileName);
+    try{const result = await tile.update({ 
+        bg_color: bg_color,
+        bg_image: bg_image,
+        hover_text: hover_text,
+        description_text: description_text,
+        link: link
+      });      
+    } catch {
+      console.log("Update failed")
+    }
+  });
+
   return (
-    <div>
-      <h2>Tile: {props.tileName}</h2>
-      <h3>Hover Text: {props.tileInfo.hover_text}</h3>
-      <h3>Description: {props.tileInfo.description_text}</h3>
-    </div>
+    <form onSubmit={onSubmit}>
+      <h2>{props.tileName}</h2>
+      <label>Color
+        <input 
+          name="bg_color"
+          type="text"
+          key={`bg_color:${props.tileInfo.bg_color}`}
+          defaultValue={props.tileInfo.bg_color}
+          ref={register}
+        />
+      </label>
+      <br />
+      <label>Image
+        <input
+          name="bg_image"
```

Comment:
Probably a question for Shannon, but do we actually intend to have background images?

Reply:
Donno, but it was a property the tiles in the db had so I included it here